src/test/java/com/example/enterprise/MoneyManagerApplicationTests.java

```diff
@@ -101,7 +101,7 @@ void checkReturnIncomeList(){
 //    Get Expense
     @Test
     void checkReturnExpensesList(){
-        expenseService.showAll();
+        expenseService.showAllExpenses();
```

Comment:
Changed the method name from showAll() to showAllExpenses() because showAll doesn't specify what to show. An outsider would not understand what it is trying to show. showAllExpenses() name is much more meaningful and provides more clarity to the code logic.